docs/en/operations/settings/settings.md

```diff
@@ -4569,3 +4569,28 @@ Type: Int64
 
 Default: 0
 
+## precise_float_parsing {#precise_float_parsing}
+
+Switches [Float32/Float64](../../sql-reference/data-types/float.md) parsing algorithms:
+* If the value is `1`, then precise method is used. It is slower than fast method, but it always returns a number that is the closest machine representable number to the input.
+* Otherwise, fast method is used (default). It usually returns the same value as precise, but in rare cases result may differ by one or two least significant bits.
```

Comment:
I think `digits` is better for DB user
```suggestion
* Otherwise, fast method is used (default). It usually returns the same value as precise, but in rare cases result may differ by one or two least significant digits.
```

Reply:
Not sure about digits, it may differ much earlier (5 digit):

```
SELECT toFloat64('1.7091')

┌─toFloat64('1.7091')─┐
│  1.7090999999999998 │
└─────────────────────┘

SELECT toFloat64(1.7091)

┌─toFloat64(1.7091)─┐
│            1.7091 │
└───────────────────┘

```

Reply:
well, it is still a mistake in a less significant digit _of the given number_ :)

Reply:
I agree, that it not different from:

```
┌─toFloat64(15008753.)─┬─toFloat64('1.5008753E7')─┐
│             15008753 │       15008753.000000002 │
└──────────────────────┴──────────────────────────┘
```

In terms of error value, but not sure that we can use word `digit` here.

 "ошибка в 6-м знаке, а какой эффект!"